ui/raidboss/data/05-shb/raid/e5n.js

```diff
@@ -144,11 +145,12 @@
         'Shock Strike': 'Schockschlag',
         'Shock(?! )': 'Entladung',
         'Lightning Bolt': 'Blitzschlag',
-        'Judgment Volts': 'Gewitter des Urteils',
+        '(?<! )Judgment Volts': 'Gewitter des Urteils',
```

Comment:
Oops! Thanks for catching these. Sorry I missed them the first time through!

Reply:
with pleasure.